src/main/java/org/websoso/WSSServer/service/PopularFeedService.java

```diff
@@ -20,4 +24,28 @@ public void createPopularFeed(Feed feed) {
         }
     }
 
+    @Transactional(readOnly = true)
+    public PopularFeedsGetResponse getPopularFeeds(User user) {
+        List<PopularFeed> popularFeeds = findPopularFeeds(user);
+        List<PopularFeedGetResponse> popularFeedGetResponses = getPopularFeedGetResponses(popularFeeds);
+        return getPopularFeedsGetResponse(popularFeedGetResponses);
```

Comment:
p5; 요 부분은 그냥 함수 사용하지 않고 그대로 쓰는게 더 잘 읽히지 않을까요?
```suggestion
        return new PopularFeedsGetResponse(popularFeedGetResponses);
```

Reply:
오호 메서드 추출하다가 잘못 들어간 것 같습니다.
반영했습니다!